linode/lke/schema_resource.go

```diff
@@ -79,6 +83,32 @@ var resourceSchema = map[string]*schema.Schema{
 					Description: "A Linode Type for all of the nodes in the Node Pool.",
 					Required:    true,
 				},
+				"tags": {
+					Type:                  schema.TypeSet,
+					Description:           "An array of tags applied to this node pool.",
```

Comment:
Super minor nitpick: 
```suggestion
					Description:           "A set of tags applied to this node pool.",

```